projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comment-like.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const displayNotice = message => {
+	const headerEnd = document.querySelector( '.wp-header-end' );
+	if ( ! headerEnd ) {
+		return;
+	}
+
+	const dismissText = window.wpcomCommentLikesData.dismiss_notice_text;
+
+	// Create notice container.
+	const notice = document.createElement( 'div' );
+	notice.className = 'notice notice-error is-dismissible';
+	// Add role="alert" so that screen readers announce the notice.
+	notice.setAttribute( 'role', 'alert' );
+
+	// Create paragraph element for the message.
+	const p = document.createElement( 'p' );
+	p.innerText = message;
+	notice.appendChild( p );
+
+	// Create dismiss (close) button.
+	const button = document.createElement( 'button' );
+	button.setAttribute( 'type', 'button' );
+	button.className = 'notice-dismiss';
+	button.setAttribute( 'aria-label', dismissText );
+
+	// Create a span element for screen readers.
+	const span = document.createElement( 'span' );
+	span.className = 'screen-reader-text';
+	span.innerText = dismissText;
+	button.appendChild( span );
+
+	// Hook up the dismiss functionality.
+	button.addEventListener( 'click', () => {
+		notice.parentNode.removeChild( notice );
```

Comment:
I think `Element#remove` is [widely available](https://developer.mozilla.org/en-US/docs/Web/API/Element/remove) now:

```suggestion
		notice.remove();
```

Reply:
Changed